src/components/PlatformOnboardingSummary/PlatformOnboardingSummary.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { Link } from '../Button'
+import { Heading, HeadingSize, Text } from '../Typography'
+
+export const PlatformOnboardingSummary = () => {
```

Comment:
Should we disable `Back` button on this step? It's right after we booked a call. And when we go back we see Calendar again. So we can disable `Back` button OR/AND show Call reminder instead

![image](https://github.com/user-attachments/assets/9775c8cb-3728-450d-9486-33f87529ae71)